unblob/cli.py

```diff
@@ -131,7 +131,7 @@ def __init__(
     "keep_extracted_chunks",
     is_flag=True,
     show_default=True,
-    help="Keep extracted chunks",
+    help="Keep extracted carved files around",
```

Comment:
Shouldn't we rename the option as well? E.g. `--keep-carved-files`

Reply:
I had been thinking about it as well. We have a 3 phrases, that are differing in a nuanced way:
- carved file
- extracted file
- extracted chunk

As far as I understand "extracted chunk" means "carved file".

I will remove this commit and make a new PR that will change both the option and the internal attribute names as well.

Reply:
Commit has been removed.